### ISSUE TITLE
Permalinks should be marked accessibly

### ISSUE THREAD

**Issue by nolanlawson** [nolanlawson/pinafore#128]
Just occurred to me that we have the little "permalink" icon (square with an arrow pointing out) but these aren't marked for ATs.

Not sure if there are ATs that already describe external links, but there may be a way to do this responsibly and unobtrusively.
